.github/workflows/golangci-lint.yml

```diff
@@ -23,3 +26,5 @@ jobs:
         uses: golangci/golangci-lint-action@v6
         with:
           version: v1.60
+          only-new-issues: true
+          args: --enable goheader
```

Comment:
Please add a comment explaining why this is "manually" enabled via flag instead of in the config.